sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/_version.py

```diff
@@ -0,0 +1,5 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+VERSION = "1.0.0b1"
```

Comment:
@swathipil Is the version needed here and in the root `_version.py` file and the extension? I assume probably not but not sure which one to keep!

Reply:
nope, this file can be removed! (+ can keep the other one)